docs/build/develop-services.md

```diff
@@ -299,4 +299,157 @@ result: "Hello, fluence"
+
+call some_service <function_name> [<arg1>, <arg2>]
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+    
+Welcome to the Marine REPL (version 0.19.1)
+Minimal supported versions
+  sdk: 0.6.0
+  interface-types: 0.20.0
+
+app service was created with service id = c3c4f345-c5e9-492c-bf58-4c2424d710c6
+elapsed time 90.803042ms
+
+1> c some_service call_curl "google.com"
+result: {
+  "error": "",
+  "ret_code": 0,
+  "stderr": "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n                                 Dload  Upload   Total   Spent    Left  Speed\n\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   219  100   219    0     0   1269      0 --:--:-- --:--:-- --:--:--  1351\n",
+  "stdout": "<HTML><HEAD><meta http-equiv=\"content-type\" content=\"text/html;charset=utf-8\">\n<TITLE>301 Moved</TITLE></HEAD><BODY>\n<H1>301 Moved</H1>\nThe document has moved\n<A HREF=\"http://www.google.com/\">here</A>.\r\n</BODY></HTML>\r\n"
+}
+ elapsed time: 198.43175ms
+
+```
+
+## Call Parameters
+Peers provide additional information about what's happening to services. This includes where the service is, who initiated the call, where are arguments from. It is known as call parameters and structure definitions are here:
+```rust
+pub struct CallParameters {
+    /// Peer id of the AIR script initiator.
+    pub init_peer_id: String,
+
+    /// Id of the current service.
+    pub service_id: String,
+
+    /// Id of the service creator.
+    pub service_creator_peer_id: String,
+
+    /// PeerId of the peer who hosts this service.
+    pub host_id: String,
+
+    /// Id of the particle which execution resulted a call this service.
+    pub particle_id: String,
+
+    /// Security tetraplets which described origin of the arguments.
+    pub tetraplets: Vec<Vec<SecurityTetraplet>>,
+}
+
+pub struct SecurityTetraplet {
+    /// Id of a peer where corresponding value was set.
+    pub peer_pk: String,
+
+    /// Id of a service that set corresponding value.
+    pub service_id: String,
+
+    /// Name of a function that returned corresponding value.
+    pub function_name: String,
+
+    /// Value was produced by applying this `json_path` to the output from `call_service`.
+    pub json_path: String,
+}
+```
+
+It is accessible through `marine_rs_sdk::get_call_parameters` function. Here is an example:
+```rust
+#[marine]
+pub fn call_parameters() -> String {
+    let cp = marine_rs_sdk::get_call_parameters();
+    format!(
+        "init_peer_id: {}, service_id: {}, service_creator_peer_id: {}, host_id: {}, particle_id: {}, tetraplets: {:?}",
+        cp.init_peer_id,
+        cp.service_id,
+        cp.service_creator_peer_id,
+        cp.host_id,
+        cp.particle_id,
+        cp.tetraplets
+    )
+}
+
+```
+
+After adding it to a service, it can be tested. Call parameters can be set manually in repl, as a second json value after arguments:
```

Comment:
It is not immediately clear to which object does "it" refer to in the sentence "After adding it to a service, it can be tested."